src/libraries/System.Private.CoreLib/src/System/Diagnostics/CodeAnalysis/StringSyntaxAttribute.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System.Diagnostics.CodeAnalysis
+{
+    /// <summary>
+    /// <see cref="StringSyntaxAttribute"/> tells an IDE the type of content that a string contains. It can use this to colorize or validate the contents of the string.
+    /// </summary>
+    /// <example>
+    ///     <code>
+    /// using System.Diagnostics.CodeAnalysis;
+    ///
+    /// public void MyRegexMethod([StringSyntax(StringSyntaxAttribute.Regex)] string myRegex) {
+    ///     // ...
+    /// }
+    ///     </code>
+    /// </example>
+    [AttributeUsage(AttributeTargets.Parameter |
+                    AttributeTargets.Field |
+                    AttributeTargets.Property)]
```

Comment:
This was approved as AllowMultiple = false, Inherited = false. The default for Inherited is true.

Reply:
Yes, but the Inherited is unnessecary on non-method or classes.

For reference VS22 message:
![grafik](https://user-images.githubusercontent.com/10373452/147145842-b8ca27d0-06db-4e61-bd6a-37ca0055f9ea.png)

& AllowMultiple = false is the default

Reply:
> For reference VS22 message:

What is producing that message?  What's the diagnostic code?

Reply:
I think its the code analyzer of `AttributeUsageAttribute`. What the diagnostics code is in the background I unfortunately can't tell you.

Reply:
The analyzer is faulty because IsInherited makes sense with AttributeTargets.Property as well. 

This does not seem to be an analyzer in dotnet/roslyn-analyzers. Do you have any Visual Studio extensions installed, that add analyzers? 

You can increase the verbosity of the build in Tools>Options>Build and then look at the build info in the Output Window and get the command line. That should include whatever this analyzer is.

Meanwhile you can ignore it/ disable this warning.

Reply:
I think that was because of R#. I have now adopted the part as in the approval.